src/app/services/errors/FileWithNameAlreadyExistsError.ts

```diff
@@ -0,0 +1,6 @@
+export class FileWithNameAlreadyExistsError extends Error {
+  static message = 'File with this name exists';
+  constructor() {
```

Comment:
Please, let's do things properly, create the error like this, respecting the prototype, the super call to pass the message and the `instanceof` usage for matching: https://github.com/internxt/drive-server-wip/blob/d040375c29e2ef806f9a6b7fd6e390cf93566c51/src/modules/user/user.usecase.ts#L51